docs/source/tools/kura-inspector.md

```diff
@@ -0,0 +1,61 @@
+# Kura Inspector
+
+Kura Inspector is a CLI tool to inspect blocks in disk storage ([Kura](../iroha_2_whitepaper.html#data-storage)).
+
+With Kura Inspector you can inspect the disk storage regardless of the operating status of Iroha and print out block contents in a human-readabe format.
+
+## Examples
+
+- Print the contents of the latest block:
+
+      kura_inspector print
+
+- Print all blocks with a height between 100 and 104:
+
+      kura_inspector -f 100 print -n 5
+
+- Print errors for all blocks with a height between 100 and 104:
+
+      kura_inspector -f 100 print -n 5 >/dev/null
+
+## Usage
+
+Run Kura Inspector:
+
+    kura_inspector [OPTIONS] <SUBCOMMAND>
+
+### Options
+
+|     Option     |                     Description                     |  Default value   |  Type   |
+| -------------- | --------------------------------------------------- | ---------------- | ------- |
+| `-f`, `--from` | The block height from which to start the inspection | The latest block | `usize` |
```

Comment:
```suggestion
| `-f`, `--from` | block height of start of range for inspection | current block height | Positive integer |
```
`usize` is a rust-specific term. While most programmers will know what it means, we want to give as much information as possible. 

Block height and block number are not the same. Block 0 corresponds to block height of 1.

Reply:
You use `usize` in `docs` to describe configs, which is why I used the same term her. 
Updated this description.